verified_sources/local_file_system/specs.yml

```diff
@@ -0,0 +1,29 @@
+description: Specification of an actor (source/generator/destination)
+type: object
+required:
+  - connection_specification
+  - name
+  - module_name
+additionalProperties: true
+properties:
+  # General Properties (Common to all connectors)
+  documentation_url:
+    type: string
+    format: uri
+  name:
+    description: "The name of the specific connector to which this ConnectorSpecification belongs."
+    type: string
+  module_name:
+    description: "Name of the python module for this connector"
+    type: string
+  connection_specification:
+    description: ConnectorDefinition specific blob. Must be a valid JSON string.
+    type: object
+    additionalProperties: true
+    properties:
+      obj_file_path:
```

Comment:
Now that you have moved the usr_file_path and obj_file_path to catalog, so they should be removed from here too

Reply:
removed from the specs.yml